kvbc/tools/sparse_merkle_db/include/sparse_merkle_db_editor.hpp

```diff
@@ -185,14 +185,33 @@ struct GetValue {
   }
 };
 
+struct RemoveMetadata {
+  std::string description() const {
+    return "removeMetadata\n"
+           "Removes metadata and state transfer data from RocksDB";
+  }
+  std::string execute(const v2MerkleTree::DBAdapter &db_adapter, const CommandArguments &) const {
+    const concordUtils::Sliver b{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::BFT),
+                                             static_cast<char>(storage::v2MerkleTree::detail::EBFTSubtype::Metadata)}};
+    const concordUtils::Sliver e{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::Key)}};
+
+    const auto status = db_adapter.getDb()->rangeDel(b, e);
+    if (!status.isOK()) {
+      throw std::runtime_error{"Failed to delete metadata and state transfer data: " + status.toString()};
+    }
+    return "Sucessfully deleted metadata and state transfer data";
```

Comment:
Nit: Since all other commands return JSON outputs, we might do the same here. Something similar to:

```
{ "result": "true" }
```

Reply:
Done